src/app/components/Buttons/CTAgreen/CTAgreen.stories.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import CTAgreen from './CTAgreen';
+
+export default {
+  title: 'Component/CTAgreen',
```

Comment:
Hmm I am having some doubts about naming since upon seeing this I have absolutely no idea what this is :D 
I´d suggest using a more descriptive name here.

Reply:
CTA means Call To Action. It is the most important Button a user should click during the user journey